atomic_reactor/plugins/check_and_set_platforms.py

```diff
@@ -76,13 +77,10 @@ def run(self) -> Optional[List[str]]:
             if is_scratch_build(self.workflow) or is_isolated_build(self.workflow):
                 override_platforms = set(user_platforms or [])
                 if override_platforms and override_platforms != set(platforms):
-                    sorted_platforms = sorted(override_platforms)
-                    self.log.info("Received user specified platforms %s", sorted_platforms)
+                    user_platforms_override = True
```

Comment:
Do we still want to consider it a user override only if it doesn't match the platforms from the koji target?

Reply:
Might not be safe to change this though, it's user-facing behavior

Reply:
brew is ALWAYS passing platforms to osbs, so this is how we are able to distinguish if user overrode platforms

Reply:
non target case is corner case when disabled koji integration

Reply:
Hmm, but this branch is the one where target _is_ defined

Reply:
koji always provides platforms to osbs

override_platforms and override_platforms != set(platforms):

means that platforms changed, aka user specified them

if they are same as target, WE can't tell if they are from target OR if user specified ALL of them

Reply:
Ah, that makes sense :+1: